ams/server/src/main/java/com/netease/arctic/server/ArcticServiceConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.netease.arctic.server;
+
+public class ArcticServiceConstants {
+
+  public static final long OPTIMIZER_CHECK_INTERVAL = 10 * 1000;
+
+  public static final long MAX_SCHEDULING_TIME = 30 * 1000;
+
+  public static final long OPTIMIZER_TOUCH_TIMEOUT = 60 * 1000;
```

Comment:
Should we make this parameter configurable, as it may be determined to timeout if there is some latency with the network.This could cause the actual running task to be removed by mistake

Reply:
Make sense.